File: src/main/java/com/AlphaStore/Core/CoreApplication.java
package com.AlphaStore.Core;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@EnableDiscoveryClient
@ComponentScan(basePackages = {
		"com.AlphaStore.Core",
		"com.AlphaStore.Utils"
})
public class CoreApplication {

	public static void main(String[] args) {
		SpringApplication.run(CoreApplication.class, args);
	}

}

File: src/main/kotlin/com/AlphaStore/Core/CoreApplication.kt
package com.alphaStore.Core

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication(
	scanBasePackages = [
		"com.alphaStore.Core",
		"com.alphaStore.Utils",
		"com.alphaStore.Admin",
		"com.alphaStore.Mail",
		"com.alphaStore.Otp"
	]
)

@EnableDiscoveryClient
class CoreApplication

fun main(args: Array<String>) {
	runApplication<CoreApplication>(*args)
}
